schematools/contrib/django/models.py

```diff
@@ -0,0 +1,317 @@
+from __future__ import annotations
+
+import re
+from typing import Any, Callable, Dict, List, Tuple, Type
+from urllib.parse import urlparse
+
+from django.contrib.gis.db import models
+from django.contrib.postgres.fields import ArrayField
+from django.db.models.base import ModelBase
+from django.conf import settings
+from django_postgres_unlimited_varchar import UnlimitedCharField
```

Comment:
Voor deze import is geen requirement in `setup.py`. Wellicht via de `extra_requires` doen?